assets/css/access.css

```diff
@@ -66,7 +66,7 @@
     font-family: 'Source Sans Pro',sans-serif;
 }
 
-.button:hover {
+.button:hover:enabled {
```

Comment:
IE 8 and older doesn't support the `:enabled` pseudo-class, so the button will look clickable even when it's disabled. Is this acceptable? If not, I'll try and find a workaround.